src/components/Compare/columns.tsx

```diff
@@ -163,29 +164,58 @@ class VaccinationsColumn extends MetricColumn {
   }
 }
 
+class HospitalizationsDensityColumn implements ColumnDefinition {
+  columnId = HOSPITALIZATIONS_DENSITY_ID;
+
+  name = 'Hospitalizations per 100k';
+
+  desiredWidthPercent = 14;
+  // Make it wide enough so progress bar doesn't look squashed on mobile.
+  minWidthPx = 130;
+
+  getValue(row: SummaryForCompare): number | null {
+    return row.metricsInfo.hd;
+  }
+
+  render(row: SummaryForCompare): React.ReactNode {
+    const hd = row.metricsInfo.hd;
+    let formattedValue = UNKNOWN_VALUE_TEXT;
+    if (hd !== null) {
+      formattedValue = hd.toFixed(1);
+    }
+
+    return (
+      <DataCellValue
+        $valueUnknown={hd === null}
+        $textAlign="left"
+        $fontSize="14px"
+      >
+        {formattedValue}
+      </DataCellValue>
+    );
+  }
+}
+
 // All of the compare table columns we use.
 const caseDensityColumn = new MetricColumn(Metric.CASE_DENSITY);
 const caseGrowthRateColumn = new MetricColumn(Metric.CASE_GROWTH_RATE);
 const positiveTestsColumn = new MetricColumn(Metric.POSITIVE_TESTS);
 const vaccinationsColumn = new VaccinationsColumn();
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const ccviColumn = new CcviColumn();
```

Comment:
Any reason we're keeping `ccviColumn` if it's unused?

Reply:
kinda seems a shame to delete all the `CcviColumn` and its related code I guess is why.

i'll remove this reference and just move the eslint disable to the `class` above actually.